zcash_keys/src/keys.rs

```diff
@@ -7,18 +8,16 @@ use zcash_primitives::{
 
 use crate::address::UnifiedAddress;
 
-#[cfg(feature = "transparent-inputs")]
-use zcash_primitives::legacy::keys::NonHardenedChildIndex;
-
 #[cfg(feature = "transparent-inputs")]
 use {
     std::convert::TryInto,
-    zcash_primitives::legacy::keys::{self as legacy, IncomingViewingKey},
+    zcash_primitives::legacy::keys::{self as legacy, IncomingViewingKey, NonHardenedChildIndex},
 };
 
 #[cfg(all(feature = "test-dependencies", feature = "transparent-inputs"))]
 use zcash_primitives::legacy::TransparentAddress;
 
+use blake2b_simd::{Params as blake2bParams, OUTBYTES as BLAKE2B_LENGTH};
```

Comment:
Nit: This import should be up at the top of the file with the other unconditional imports from third-party crates.